src/components/dls/QuranWord/GlyphWord.tsx

```diff
@@ -51,22 +50,23 @@ const GlyphWord = ({
   font,
   isFontLoaded,
 }: UthmaniWordTextProps) => {
-  const quranReaderStyles = useSelector(selectQuranReaderStyles, shallowEqual);
-  const { quranTextFontScale, mushafLines } = quranReaderStyles;
+  const { quranTextFontScale, mushafLines } = useSelector(selectQuranReaderStyles, shallowEqual);
+  const innerHTML = {
+    __html: ` ${getWordText(qpcUthmaniHafs, textCodeV1, textCodeV2, font, isFontLoaded)}`,
+  };
```

Comment:
Adding the extra spacing before the word broke V2 Mushaf's layout  since `GlyphWord` is common for all browsers and also for all Glyph-based Mushafs.
Before:
<img width="554" alt="Screenshot 2024-05-28 at 3 56 13 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/74952381-b412-4bda-ad4b-d5c7de48323f">

After:
<img width="626" alt="Screenshot 2024-05-28 at 3 55 58 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/f6e43850-f067-4296-bb37-9010ac115f18">